backend/src/test/java/com/htecgroup/skynest/utils/BucketEntityUtil.java

```diff
@@ -18,4 +21,12 @@ public static BucketEntity getPrivateBucket() {
     bucketEntity.setCreatedBy(UserEntityUtil.getVerified());
     return bucketEntity;
   }
+
+  public static BucketEntity getOtherPrivateBucket() {
+    BucketEntity bucketEntity = getPrivateBucket();
+    UserEntity userEntity = UserEntityUtil.getVerified();
+    userEntity.setId(UUID.randomUUID());
```

Comment:
didn't you say random uuids in tests were a problem?

Reply:
This is special case where I need to return some bucket with other owner than the loggedUser, so it's implemented in this method. Same UUID is in other used util classes so I need other to check when user is trying to edit others bucket.